sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -290,22 +295,25 @@ def message(self, value: "Message") -> None:
 
     @property
     def raw_amqp_message(self) -> AmqpAnnotatedMessage:
-        """Advanced usage only. The internal AMQP message payload that is sent or received."""
+        """Advanced usage only. The internal AMQP message payload that is sent or received.
+
+        :rtype: ~azure.eventhub.amqp.AmqpAnnotatedMessage
```

Comment:
wont this need an accompanying return ?

Reply:
@property classes only require an rtype :)

Reply:
Oh yes I remember it from the other issue now